docs/docker-dev/docker-dev-setup-manual.md

```diff
@@ -1,64 +1,77 @@
 # Docker Development Environment Setup
+
 The following instructions demonstrate how to set up a development environment for OpenSearch Dashboards using Docker. It utilizes tools such as `Docker` and `VS Code`, and users should be familiar with the basic usages of them. Users will be able to develop and run the application inside VS Code without additional configurations.
 
 1. Install [Docker](https://docs.docker.com/get-docker/) if not already installed.
-    * Make sure that Docker daemon is running. (For windows and macos，you need to have [Docker Desktop](https://docs.docker.com/desktop/), or its alternatives, such as [Finch](https://github.com/runfinch/finch))
+
+   - Make sure that Docker daemon is running. (For windows and macos，you need to have [Docker Desktop](https://docs.docker.com/desktop/), or its alternatives, such as [Finch](https://github.com/runfinch/finch))
 
 2. In the terminal, run the command below.
-    * This should create a folder named `opensearch-dashboards-docker-dev` and it should contain two files: `docker-compose.yml` and `entrypoint.sh`.
-    * Here is the link to the installer script: `https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/main/dev-tools/install-docker-dev.sh` if needed.
+   - This should create a folder named `opensearch-dashboards-docker-dev` and it should contain two files: `docker-compose.yml` and `entrypoint.sh`.
+   - Here is the link to the installer script: `https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/main/dev-tools/install-docker-dev.sh` if needed.
```

Comment:
nit: We can use a relative path link here. That way for folks reading this in a local copy of the repo in their code editor, they'll view the file from local in their editor as opposed to opening the GitHub page. It'll still navigate to the GitHub page for folks viewing there.

Ex.
```
... [installer script](/dev-tools/install-docker-dev.sh)
```